vendor/github.com/fsamin/go-dump/encoder.go

```diff
@@ -231,7 +231,9 @@ func (e *Encoder) ToStringMap(i interface{}) (res map[string]string, err error)
 	}
 	res = map[string]string{}
 	for k, v := range ires {
-		res[k] = fmt.Sprintf("%v", v)
```

Comment:
does it need a PR on go-dump ?

Reply:
yes, doing

Reply:
PR: https://github.com/fsamin/go-dump/pull/14

Reply:
merged, you can update the vendor

Reply:
done